.env.template

```diff
@@ -13,6 +13,12 @@
 ## AI_SETTINGS_FILE - Specifies which AI Settings file to use (defaults to ai_settings.yaml)
 # AI_SETTINGS_FILE=ai_settings.yaml
 
+## OPENAI_API_BASE_URL - Custom url for the OpenAI API, useful for connecting to custom backends. No effect if USE_AZURE is true, leave blank to keep the default url 
+# OPENAI_API_BASE_URL=
```

Comment:
Should this be openai base url or LLM base url?

Reply:
since this is specifically to redirect openai-like traffic to a local or different host, i would say OPEN_API_BASE_URL is appropriate here!

Reply:
Yeah, i see "OPENAI_API_BASE_URL" as more appropriate because it still uses OpenAI's API